kernel/packages/shared/comms/actions.ts

```diff
@@ -29,3 +29,7 @@ export type VoiceRecordingUpdate = ReturnType<typeof voiceRecordingUpdate>
 export const SET_VOICE_VOLUME = 'Set Voice Volume'
 export const setVoiceVolume = (volume: number) => action(SET_VOICE_VOLUME, { volume })
 export type SetVoiceVolume = ReturnType<typeof setVoiceVolume>
+
+export const SET_VOICE_MUTE = 'Set Voice Mute'
+export const setVoiceMute = (mute: boolean) => action(SET_VOICE_MUTE, { mute })
```

Comment:
Could it be that this action is not used? Is there something missing?

Reply:
Is not used... I added it in a first approach and then decided to keep it just in case